vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -37,7 +37,7 @@ $global:defaultContainerdPackageUrl = "https://acs-mirror.azureedge.net/containe
 
 $global:defaultDockerVersion = "20.10.9"
 
-switch ($windowsSKU) {
+switch -Regex ($windowsSKU) {
     "2019" {
```

Comment:
Did you test your changes? 2019-containerd will also fit "2019" with regex switch

Reply:
Need to inspect the logs. Yes, I tested four cases.

Reply:
Outdated. Replaced by "if else" statement clauses.

Reply:
Checked the pulled images in pipeline logs. `pulling images for ****` I think it's now pulling the correct images per `windowsSku`. Are there any other issues?